### ISSUE TITLE
Missing parent GO:0101005   deubiquitinase activity

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#23251]
GO:0101005   deubiquitinase activity
Molecular Function
is defined
An **isopeptidase activity** that cleaves ubiquitin from a target protein to which it is conjugated.

but it is not currently a descendant of 
GO:0070122 | isopeptidase activity

source: iEA mapping to GO:0070122 | isopeptidase activity here:
https://www.pombase.org/gene/SPAC31G5.13
is not suppressed as redundant.

**Comment by pgaudet** [comment]
Added 'ubiquitin-like protein peptidase activity' is_a isopeptidase activity. 
This should be inherited by GO:0101005 deubiquitinase activity


@sjm41 I forget why we didn't do that. I think we were debating whether 'isopeptidase' was a useful term, s as it is not orthogonal to other terms in that branch. 

Thanks, Pascale

**Comment by ValWood** [comment]
I would be happy for 'isopeptidase' to go, there are too many axes of classification for peptidases!

**Comment by pgaudet** [comment]
Actually I found it: One of the children of GO:0101005 deubiquitinase activity is 'GO:0140492 metal-dependent deubiquitinase activity' 
"Catalysis of the metal-dependent hydrolysis of an ester, thioester, amide, peptide or isopeptide bond formed by the C-terminal glycine of ubiquitin.

See discussion in #17398


Something is wrong because 'deubiquitinase activity' should be named 'deubiquitinase isopeptide activity' and the metallo-dependent version should not be a child.

**Comment by pgaudet** [comment]
Actually we could create 'deubiquitinase isopeptide activity' and give it the correct parent, and keep GO:0101005 deubiquitinase activity  as the grouping term. Does that work?

**Comment by sjm41** [comment]
Hi both

1. My memory from our final #17398 discussions was that we were planning to obsolete the 'isopeptidase activity' GO term - there are only 6 EXP annotations, and 5 of those are to the (current) single child term "leukotriene-C(4) hydrolase" (and I'm not sure that is correctly classified anyway.... I'll make a separate ticket for that.)

2. I'm not sure the definition on 'GO:0140492 metal-dependent deubiquitinase activity' is correct - I don't easily see any statement about metal-dependent deubiquitinases working on non-isopeptide bonds in the given ref (PMID:19489724). I think the def just needs to model the def for GO:0101005 deubiquitinase activity, with 'metal-dependent' added.

3. FYI, we added this comment to GO:0019783 ubiquitin-like protein peptidase activity "While ubiquitin-like proteins can be rarely linked to substrates via bonds other than isopeptide bonds, all known ubiquitin-like peptidases cleave the isopeptide bond."

So, if we proceed with obsoleting 'isopeptidase activity' (as an unnecessary/complicating additional classification axis) then Val's reported issue goes away, and we don't need to make any new grouping terms here.
